iam.tf

```diff
@@ -52,4 +57,30 @@ data "aws_iam_policy_document" "claimant_api_kafka_consumer" {
     ]
     resources = [aws_cloudwatch_log_group.claimant_api_kafka_consumer.arn]
   }
+
+  statement {
+    sid       = "AllowSSMGetNinoSalt"
+    effect    = "Allow"
+    actions   = ["ssm:GetParameter"]
+    resources = [data.aws_ssm_parameter.nino_salt_london_ssm_param.arn, ]
+  }
+
+  statement {
+    sid     = "AllowKMSDecryptNinoSalt"
+    effect  = "Allow"
+    actions = ["kms:Decrypt", ]
```

Comment:
For neatness close off the actions without a comma. Possibly think that a service is missing.

Reply:
I like to always have commas at the end - makes you less prone to forget them. Taste thing I guess.